tests/Integration/Behaviour/Features/Context/Domain/CategoryFeatureContext.php

```diff
@@ -456,6 +457,27 @@ public function bulkDisableCategories(string $categoriesReferences)
         $this->getCommandBus()->handle(new BulkUpdateCategoriesStatusCommand($categoryIds, false));
     }
 
+    /**
+     * @Given category :categoryReference in default language named :categoryName exists
+     *
+     * @param string $categoryReference
+     * @param string $categoryName
+     */
+    public function assertCategoryExistsByName(string $categoryReference, string $categoryName)
+    {
+        $foundCategory = Category::searchByName($this->defaultLanguageId, $categoryName, true);
+
+        if (!$foundCategory || $foundCategory['name'] !== $categoryName) {
```

Comment:
```suggestion
        if (!isset($foundCategory['name']) || $foundCategory['name'] !== $categoryName) {
```

You should check for `$foundCategory['name']` which is the key you're using in the condition.